arm-batch/2017-01-01/swagger/BatchManagement.json

```diff
@@ -1150,10 +1151,15 @@
         "id": {
           "type": "string",
           "description": "The resource ID of the Azure key vault associated with the Batch account."
+        },
+        "url": {
+          "type": "string",
```

Comment:
Given this is a request parameter, it would make sense to add constraints like minLength, maxLength and a regex. At least a regex is highly recommended for the url string